### ISSUE TITLE
Automate some or all file syncs from this repo

### ISSUE THREAD

**Issue by jezdez** [conda/infrastructure#589]
### Checklist

- [X] I added a descriptive title
- [X] I searched open requests and couldn't find a duplicate

### What is the idea?

I think we should for some files fully automate the file-syncing between the infra repo and the various other repos, since it's becoming an issue to merge all those PRs.

### Why is this needed?

To stay effective with the file-syncing from the infra repo, we should stop using the pull-request based workflow and sync some of the files fully automatically.

### What should happen?

Less time spent on merging PRs in the various conda repos that just sync files from this repo.

**Comment by kenodegard** [comment]
I don't think this is possible UNLESS we adjust the conda-bot to have push access to the main branches, per the sync workflow the [`SKIP_PR` option](https://github.com/BetaHuhn/repo-file-sync-action):

> Skips creating a Pull Request and pushes directly to the default branch

**Comment by kenodegard** [comment]
This may be possible to do if we add the conda bot user as a special user that can push to the main branch and bypass checks, dunno if we're comfortable with this from a security perspective @awwad 

<img width="787" alt="Screen Shot 2022-09-09 at 9 15 13 AM" src="https://user-images.githubusercontent.com/4546435/189371010-3312fe6a-6e00-4ea0-b87e-c06867e8f058.png">
<img width="781" alt="Screen Shot 2022-09-09 at 9 15 30 AM" src="https://user-images.githubusercontent.com/4546435/189371014-f75f0674-c77c-497b-adc7-92807b986e06.png">
